packaging/installer/README.md

```diff
@@ -293,6 +293,28 @@ To apply the changes you made, you have to restart Netdata.
 
 ---
 
+### Binary Packages
+![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
+
+We provide our own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB packaging formats.
+We have currently released .RPM versions with version [1.16.0](https://github.com/netdata/netdata/releases/tag/v1.16.0). We are planning to release packages following the .DEB format on one of the following releases. Our current packaging infrastructure provider is [Package Cloud](https://packagecloud.io).
+Netdata is committed to support installation of our solution to all operating systems.
```

Comment:
Start a new paragraph here, and combine the next two lines into this paragraph.

Reply:
Sorry, the code blocks aren't really working right. Start a new paragraph at `We have currently released...`